prem_utils/connectors/prem.py

```diff
@@ -15,114 +20,36 @@ def __init__(
         self,
         api_key: str,
         base_url: str = "https://ml-development.prem.ninja/api/ml",
+        prem_ml_url: str = "https://ml.premai.io/api/ml/slm-completion/",
```

Comment:
I've missed this. Both should point to ml.premai.io, there is no dev anymore.  Also, I would only keep the base_url and handle the slm_completion url internally

Reply:
Updated